src/connection/session-manager.js

```diff
@@ -21,17 +22,29 @@ class SessionManager {
     }
   }
 
-  getId() {
+  getSessionId() {
     return this._getStorage().getItem(STORAGE_SESSION_NAME)
   }
 
-  setId(value) {
+  getExternalId() {
+    return this._getStorage().getItem(STORAGE_EXTERNAL_ID)
+  }
+
+  setSessionId(value) {
     return this._getStorage().setItem(STORAGE_SESSION_NAME, value)
   }
 
-  clear() {
```

Comment:
I think we can keep the `clear()` method which will always clear everything. There shouldn't be any case where you want to clear one key but keep the other